src/REPLICA/fix_pimd_langevin.cpp

```diff
@@ -514,10 +514,15 @@ void FixPIMDLangevin::init()
 void FixPIMDLangevin::setup(int vflag)
 {
   int nlocal = atom->nlocal;
+  int *mask = atom->mask;
   double **x = atom->x;
   imageint *image = atom->image;
   if (mapflag) {
-    for (int i = 0; i < nlocal; i++) domain->unmap(x[i], image[i]);
+    for (int i = 0; i < nlocal; i++) {
```

Comment:
There is no actual change in this function, the original code should be restored and the comments removed.